trestle/transforms/implementations/osco.py

```diff
@@ -28,14 +28,17 @@
 from ruamel.yaml import YAML
 
 from trestle.common.list_utils import as_list
-from trestle.oscal.assessment_results import ControlSelection
 from trestle.oscal.assessment_results import LocalDefinitions1
 from trestle.oscal.assessment_results import Observation
 from trestle.oscal.assessment_results import Result
-from trestle.oscal.assessment_results import ReviewedControls
-from trestle.oscal.assessment_results import Status1
 from trestle.oscal.assessment_results import SystemComponent
-from trestle.oscal.common import ImplementedComponent, InventoryItem, Property, SubjectReference
+from trestle.oscal.common import ControlSelection
```

Comment:
A really silly/minor thing, why are these imports separated instead of being a one liner? Any particular reason even though they are coming from the same class?

Reply:
fixed.